std/traits.d

```diff
@@ -388,7 +388,44 @@ version (none) version(unittest) //Please uncomment me when changing packageName
 }
 
 /**
- * Get the module name (including package) for the given symbol.
+ * Get the module name (including package) for the given symbol,
+ * returning it as a string.
+ *
+ * Example:
+ *    This can be used to implement something akin to
+ *    $(LINK2 https://en.wikipedia.org/wiki/Argument-dependent_name_lookup, Argument Dependent Lookup):
```

Comment:
Please replace `LINK2 https://en.wikipedia...` with `HTTPS en.wikipedia...`. The colon in the link is interpreted wrongly by ddoc when the paragraph gets reflowed in maintenance.